packages/eslint-plugin/package.json

```diff
@@ -98,12 +98,8 @@
   },
   "peerDependencies": {
     "@typescript-eslint/parser": "^8.0.0 || ^8.0.0-alpha.0",
-    "eslint": "^8.57.0 || ^9.0.0"
-  },
-  "peerDependenciesMeta": {
-    "typescript": {
-      "optional": true
-    }
+    "eslint": "^8.57.0 || ^9.0.0",
+    "typescript": ">=4.2.0"
```

Comment:
This is not the correct range.
Please mirror the range in the root package.json

Reply:
`ts-api-utils` doesn't support `typescript` < 4.2.0 (`"typescript": "*"` can't be used)

Reply:
I meant this range.

https://github.com/typescript-eslint/typescript-eslint/blob/88e4c66874656ac78ab50d4ba2c5f7d1b999bab9/package.json#L123

`>=4.2.0` is entirely INCORRECT as a dependency -- we DO NOT support TS 4.2.0
Similarly we DO NOT support TS `>=5.7.0`.

Reply:
Thanks, I did the commit